turbopack/crates/turbopack-browser/src/ecmascript/content_entry.rs

```diff
@@ -52,10 +52,23 @@ impl EcmascriptDevChunkContentEntries {
     ) -> Result<Vc<EcmascriptDevChunkContentEntries>> {
         let chunk_content = chunk_content.await?;
 
-        let entries: FxIndexMap<_, _> = chunk_content
+        let included_chunk_items = chunk_content
             .chunk_items
             .iter()
-            .map(|&(chunk_item, async_module_info)| async move {
+            .map(|(ty, item, async_info)| async move {
+                if matches!(*ty.await?, ChunkItemTy::Included) {
+                    Ok(Some((item, async_info)))
+                } else {
+                    Ok(None)
+                }
+            })
+            .try_join()
+            .await?
+            .into_iter()
+            .flatten();
```

Comment:
This would be better if you split into `chunk_items` and `passthrough_items` and avoid the `ChunkItemTy` enum